client/performance-profiler/pages/loading-screen/progress.tsx

```diff
@@ -115,18 +117,30 @@ const useLoadingSteps = ( {
 	if ( isLoadingPages ) {
 		steps = [ translate( 'Getting your site pages' ) ];
 	} else {
-		steps = isSavedReport
+		const loggedInSteps = [
+			...( step === 0
+				? [ translate( 'Checking for existing report…' ) ]
```

Comment:
I don’t think we use "report" anywhere else in the copy, so let’s tweak these messages to:

> Checking for recent performance tests

> No recent tests found